source/howto/mobility/getstarted.rst

```diff
@@ -0,0 +1,59 @@
+How to Access Mobility Data With API v2
+=======================================
+
+The new :strong:`API v2` (see :ref:`the description <ninja api>` for
+the Mobility domain has simplified the access to data; among its
+features, we recall that there is now one single endpoint from which
+to retrieve data from all datasets.
+
+The starting point for all actions to be carreid out on the datasets
+made available by the |odh| team is the following:
+
+https://mobility.api.opendatahub.bz.it/v2/swagger-ui.html
+
+.. figure:: /images/mobility-swagger.png
+
+   The swagger interface of the Mobility API v2.
+
+From this site, links provide access to documentation about data
+licencing and use of the API; it is also possible to contact the |odh|
+team by sending an email to the issue tracker, to ask questions,
+provide feedback, or to report issues.
+
+Getting Started
+---------------
+
+In the API v2, the central concept is :strong:`Station`: all data come
+from a given |station|\, whose complete list can be
+retrieved by simply opening the first method of the
+:strong:`data-controller`, :strong:`/api`, then click on :button:`Try
+it out` and then on `Execute`.
```

Comment:
@stefanodavid Hi, I suggest to talk about StationTypes here, since Stations are single nodes that collect data, but StationTypes are a set of Stations that a related

Reply:
Hi @Piiit, please see my comment in PR #100